src/lib/events.js

```diff
@@ -98,10 +120,24 @@ var Events = {
         /*
          * Call all the handlers except the last one.
          */
-        for(var i = 0; i < handlers.length; i++) {
+        for(i = 0; i < handlers.length; i++) {
             handlers[i](data);
         }
 
+        /* Do the same as for external-facing events, except trigger the same
+         * events on the internal handlers. This does *not* affect the return
+         * value. It simply mirrors triggers internally so that there's no
+         * conflict with external user-defined events when plotly manages
+         * events internally.
+         */
+        var internalHandlers = plotObj._internalEv._events[event];
```

Comment:
Here's the only reason (that I can think of) why we still have that trigger event block: https://plot.ly/javascript/hover-events/#triggering-hover-events 

I'd vote for dropping it in `v2.0.0` (cc @alexcjohnson)

So, I'd vote for not adding support for `triggerHandlers` for internal events?